pkgs/applications/file-managers/ranger/default.nix

```
@@ -1,28 +1,30 @@
-{ lib, fetchFromGitHub, python3Packages, file, less, highlight, w3m, ranger, testers
+{ lib, fetchFromGitHub, python3Packages, file, less, highlight, w3m, ranger, imagemagick, testers
 , imagePreviewSupport ? true
+, sixelPreviewSupport ? true
 , neoVimSupport ? true
 , improvedEncodingDetection ? true
 , rightToLeftTextSupport ? false
 }:
 
 python3Packages.buildPythonApplication rec {
   pname = "ranger";
-  version = "1.9.3";
+  version = "unstable-2023-08-23";
```

```suggestion
  version = "1.9.3-unstable-2023-08-23";
```

https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming:

> If a package is a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by -unstable- and the date of the (fetched) commit. The date must be in "YYYY-MM-DD" format.

> Example: Given a project had its latest releases 2.2 in November 2021, and 3.0 in January 2022, a commit authored on March 15, 2022 for an upcoming bugfix release 2.2.1 would have version = "2.2-unstable-2022-03-15".

---

@9ary, can you incorporate this suggestion? (Add `Co-authored-by:` if you don't commit it via the GitHub UI.)

---

I'll do it later

---

Sorry, I forgot about this.

I wasn't aware of that convention (I don't think I've actually seen it in use before). Fixing this now.